src/app/core/tools/WiringTool.ts

```diff
@@ -91,8 +93,14 @@ export const WiringTool = (() => {
 
             const port2 = findPort(info);
             if (port2 !== undefined) {
-                const wire = CreateWire(circuit.getWireKind(), port!.id, port2.id);
-                history.add(Place(circuit, wire));
+                if (port2.group === DigitalPortGroup.Output) {
```

Comment:
Gotta get back up-to-date with `model_refactor` since groups are now strings
Also add a TODO at the top saying this shouldn't be necessary in the future hoepfully